apps/db/test/support/factory.ex

```diff
@@ -118,6 +118,7 @@ defmodule DB.Factory do
       target_user: build(:user),
       type: :create,
       entity: :comment,
+      comment: build(:comment),
```

Comment:
This is what make the test fail. Is it really required?

Reply:
For my test yes, without I doesn't have the proper data in my db. But it's maybe not required here.